FiberSpectrograph/v4/_init.yaml

```diff
@@ -0,0 +1,2 @@
+s3instance: test
+image_service_url: "http://comcam-mcm.tu.lsst.org"
```

Comment:
I suggest a location value here such as `location: unknown`

because otherwise we cannot run this anywhere except summit and tucson -- i.e. we can't run simulated versions at the base.